db/migrate/20220210060921_create_tasks.rb

```diff
@@ -3,7 +3,7 @@ def change
     create_table :tasks do |t|
       t.string :title
       t.string :discription
-      t.datetime :started_at
+      t.string :started_at
       t.string :close_at
```

Comment:
please make it closed_at.
Shouldn't it be a datetime?

Reply:
create_task migration is already on production. Add new migration to update column.